83 button outline graidant

SKupferstein opened the pull request.

leowald reviewed (changes requested):
I am very impressed by this..
What I want you to do now is clean up the code.
All the css - except for what needs to be (like background color) should be in the css file.
See what you can move out, while still keeping the implementation the same.

SKupferstein commented:
I added two pushes by mistake!!
Please ignore
Sorry

On Tue, 24 Dec 2024 at 11:29, leowald ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> I am very impressed by this..
> What I want you to do now is clean up the code.
> All the css - except for what needs to be (like background color) should
> be in the css file.
> See what you can move out, while still keeping the implementation the same.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/leowald/yeshtikvah/pull/87#pullrequestreview-2521729840>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFNBRM26JYHPEM2AMKK45WL2HESPJAVCNFSM6AAAAABUEMUPUKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKMRRG4ZDSOBUGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

leowald commented:
Funny because I just went to check it and was confused because I didn't see any changes :)


> I added two pushes by mistake!! Please ignore Sorry
> […](#)
> On Tue, 24 Dec 2024 at 11:29, leowald ***@***.***> wrote: ***@***.**** requested changes on this pull request. I am very impressed by this.. What I want you to do now is clean up the code. All the css - except for what needs to be (like background color) should be in the css file. See what you can move out, while still keeping the implementation the same. — Reply to this email directly, view it on GitHub <[#87 (review)](https://github.com/leowald/yeshtikvah/pull/87#pullrequestreview-2521729840)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BFNBRM26JYHPEM2AMKK45WL2HESPJAVCNFSM6AAAAABUEMUPUKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKMRRG4ZDSOBUGA> . You are receiving this because you authored the thread.Message ID: ***@***.***>

leowald reviewed (changes requested):
Excellent. looks great.
Here are a few small changes:
A test fails - please figure out why
Storybook - theme add none (you had this originally and you are write you need it) to go back
There are more files here pushed with changes then the ones you changed. The only differences here should be  the files you changed - no others.  So only Button.jsx, Button.modules.scss, Button.stories.jsx, Buttons.test.jsx

SKupferstein commented:
Pushed again
Regarding different files being changed - are you able to tell me which
files?

On Sun, 29 Dec 2024 at 11:02, leowald ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Excellent. looks great.
> Here are a few small changes:
> A test fails - please figure out why
> Storybook - theme add none (you had this originally and you are write you
> need it) to go back
> There are more files here pushed with changes then the ones you changed.
> The only differences here should be the files you changed - no others. So
> only Button.jsx, Button.modules.scss, Button.stories.jsx, Buttons.test.jsx
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/leowald/yeshtikvah/pull/87#pullrequestreview-2524906742>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFNBRMYTCLKSFILLZIHUBX32H63CVAVCNFSM6AAAAABUEMUPUKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKMRUHEYDMNZUGI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

leowald commented:
go to https://github.com/leowald/yeshtikvah/pull/87/files and you will see.

SKupferstein commented:
OK
I pushed again:
fixed 2 things
otherwise there were 2 mdx files that got recreated when I ran Storybook
1 thing I added in the icon component - there was a mistake there that
caused me problems in the button component, so I fixed it.

On Sun, 29 Dec 2024 at 21:58, leowald ***@***.***> wrote:

> go to https://github.com/leowald/yeshtikvah/pull/87/files and you will
> see.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/leowald/yeshtikvah/pull/87#issuecomment-2564825273>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFNBRM4GWIZB6TTSNK4I2TD2IBH7XAVCNFSM6AAAAABUEMUPUKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNRUHAZDKMRXGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
